### PR TITLE
Track settings pages usage

### PR DESCRIPTION
**Pull request by remidej**
### What does it do?

Adds a `willNavigate` telemetry event when a user clicks on a link in the settings menu

### Why is it needed?

To track the usage of the audit logs page for users who did not disable telemetry, and offer that possibility for other settings pages as well

### How to test it?

- Open the settings page
- Make sure your browser's devtools are open and on the network page
- Click on the audit logs link
- You should see a "track" request to `https://analytics.strapi.io/api/v2/track`, its payload should be an object with an event property of `willNavigate`, and an eventProperties object with `/settings/audit-logs?pageSize=50&page=1&sort=date:DESC` as the destination

### CODE REVIEW

File: packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js [hunk 1]
```diff
@@ -35,14 +37,26 @@ const SettingsNav = ({ menu }) => {
     defaultMessage: 'Settings',
   });
 
+  const handleClick = (link) => {
+    if (link.to.startsWith('/settings/audit-logs')) {
+      trackUsage('didGoToAuditLogs');
+    }
+  };
+
```
Comment (gu-stav) [inline packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js:45]: Could you circle this back and check if we can use 

```
trackUsage('willNavigate', { from: pathname, to: destination });
```

here too? This was introduced in https://github.com/strapi/strapi/pull/14875/files and at the time we decided we don't want individual events for every item, but one event instead that gives us deep insight into navigations.

**Review by gu-stav — CHANGES_REQUESTED** [review]

File: packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js (hunk 1 above)
Comment (remidej) [reply packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js]: Approved by Yannis 👍

File: packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js [hunk 2]
```diff
@@ -35,14 +38,24 @@ const SettingsNav = ({ menu }) => {
     defaultMessage: 'Settings',
   });
 
+  const handleClickOnLink = (destination = null) => {
```
Comment (markkaylor) [inline packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js:41]: just curious why you set a value here? If nothing is passed won't it just be null / undefined?

**Review by markkaylor — APPROVED** [review]
Just a small question but it's probably fine as is

File: packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js (hunk 2 above)
Comment (remidej) [reply packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js:41]: Not really sure why it's required either, I just thought I'd copy the [exact implementation](https://github.com/strapi/strapi/blob/430461f493de4210c3d73f42f7d09a5d34eb3d24/packages/core/admin/admin/src/components/LeftMenu/index.js#L98-L101) of the LeftMenu. Maybe it makes a difference in Amplitude whether it's null or undefined? 🤷

Comment (markkaylor) [reply packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js:41]: 10-4 works for me 🎈

**Pull request merged**